### PR TITLE
phpstan: Streamline vendor file location with local dev-env

### PR DESCRIPTION
**Pull request by sukhwinder33445**

### CODE REVIEW

**Review by raviks789 — APPROVED** [review]

File: .github/workflows/php.yml
Comment (nilmerg) [inline .github/workflows/php.yml:1]: PHP 8.3 is missing in the lint's job matrix

File: library/Businessprocess/Common/Sort.php [hunk 1]
```diff
@@ -41,7 +41,9 @@ public function setSort(?string $sort): self
             return $this;
         }
 
-        list($sortBy, $direction) = Str::symmetricSplit($sort, ' ', 2, 'asc');
+        /** @var array<int, string> $res */
```
Comment (nilmerg) [inline library/Businessprocess/Common/Sort.php:44]: This must be fixed in ipl-stdlib.

File: phpstan-baseline-standard.neon [hunk 2]
```diff
@@ -3670,6 +3570,11 @@ parameters:
 			count: 1
 			path: library/Businessprocess/Storage/LegacyStorage.php
 
+		-
+			message: "#^Method Icinga\\\\Module\\\\Businessprocess\\\\Storage\\\\Storage\\:\\:__construct\\(\\) has parameter \\$config with generic class Icinga\\\\Data\\\\ConfigObject but does not specify its types\\: TValue$#"
```
Comment (nilmerg) [inline phpstan-baseline-standard.neon:3574]: This is related to a recent change in Icinga Web and should be properly fixed.

File: phpstan-baseline-standard.neon [hunk 3]
```diff
@@ -3710,6 +3615,11 @@ parameters:
 			count: 1
 			path: library/Businessprocess/Storage/Storage.php
 
+		-
+			message: "#^Property Icinga\\\\Module\\\\Businessprocess\\\\Storage\\\\Storage\\:\\:\\$config with generic class Icinga\\\\Data\\\\ConfigObject does not specify its types\\: TValue$#"
```
Comment (nilmerg) [inline phpstan-baseline-standard.neon:3619]: This is related to a recent change in Icinga Web and should be properly fixed.

File: phpstan-baseline-standard.neon (hunk 2 above)
Comment (sukhwinder33445) [reply phpstan-baseline-standard.neon:3574]: Fixed in https://github.com/Icinga/icingaweb2-module-businessprocess/pull/446

File: phpstan-baseline-standard.neon (hunk 3 above)
Comment (sukhwinder33445) [reply phpstan-baseline-standard.neon:3619]: Fixed in https://github.com/Icinga/icingaweb2-module-businessprocess/pull/446

File: .github/workflows/php.yml
Comment (sukhwinder33445) [reply .github/workflows/php.yml:1]: Added in https://github.com/Icinga/icingaweb2-module-businessprocess/pull/446

**Pull request merged**